.gitlab-ci.yml

```diff
@@ -96,17 +96,17 @@ build:native:
   needs: []
   stage: test
 
-test:packages:
+check:packages:
```

Comment:
These used to be named `check` when I created them, I named them that again because with more and more test scripts it's otherwise hard to see which ones are actually running tests and which are just checking something else